docs/pages/templates/chat/index.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Chat'
+---
+
+import { Cards } from 'nextra-theme-docs'
+import { Card } from '../../../components/Card'
+
+# Chat
+
+The chat template is a fully fledged chat application showcase the uses of the wallet-to-wallet messaging feature of the Liteflow infrastructure.
+
+<Cards num={2}>
+  <Card noPadding>![Home](/templates/chat/home.png)</Card>
+  <Card noPadding>![Home](/templates/chat/conversation.png)</Card>
+  <Card noPadding>![Home](/templates/chat/embed.png)</Card>
+  <Card noPadding>![Home](/templates/chat/widget.png)</Card>
+</Cards>
+
```

Comment:
### Open Source
This application is open-source and fork-able. You can view the source code and contribute to it on: https://github.com/liteflow-labs/chat